firefox-ios/Client/Frontend/Share/ShareExtensionHelper.swift

```diff
@@ -17,19 +17,8 @@ class ShareExtensionHelper: NSObject, FeatureFlaggable {
     private let pocketIconExtension = "com.ideashower.ReadItLaterPro.AddToPocketExtension"
     private let pocketActionExtension = "com.ideashower.ReadItLaterPro.Action-Extension"
 
-    var areShareSheetChangesEnabled: Bool {
-        return featureFlags.isFeatureEnabled(.shareSheetChanges, checking: .buildOnly) && !url.isFileURL
-    }
-
-    /// Exclude 'Add to Reading List' which currently uses Safari. If share sheet changes are enabled exclude
-    /// Copy from system to provide custom activity
```

Comment:
@adudenamedruby this comment was not clear to me lol, but I think this logic is right?